keylime/cloud_verifier_common.py

```
@@ -187,6 +187,25 @@ def process_quote_response(
     tenant_keyring = file_signatures.ImaKeyring.from_string(verification_key_string)
     ima_keyrings.set_tenant_keyring(tenant_keyring)
 
+    # Warn when accept-all ignores a non-empty measured boot reference state.
+    # This warning is intentionally emitted only once per agent attestation lifetime.
+
+    if mb_policy_name == "accept-all" and mb_policy:
+        try:
+            parsed_mb = json.loads(mb_policy)
+            has_nonempty_refstate = bool(parsed_mb) if isinstance(parsed_mb, dict) else True
+        except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)